### ISSUE TITLE
Add ability to show name of Basemap in attribution 

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/terriajs#6469]
Currently the Basemap attribution doesn't show the name of the Basemap for which that attribution belongs to. This creates confusion on data displayed and who is attributed to and why. 
The plan is to show the same name in the Basemap link at the bottom of landing page as the one shown in Map settings when hoovering over the selection of Basemaps. 

New title when opening Basemap attribution link: 
"Data for Basemap [insert name[ provided by:"

![image](https://user-images.githubusercontent.com/18452161/182300767-ac7d3edc-e284-404a-bf3b-bcd7dcaa4c6f.png)

![image](https://user-images.githubusercontent.com/18452161/182300878-26c29199-1e9d-42eb-9edc-9588cf3774b3.png)

**Comment by zoran995** [comment]
Won't this change cause more confusion? Data attribution for long time is not only for base maps but for all data available on the map. 
Also, if I recall correctly, when there are multiple attributions for same data provider (multiple datasets) they will be grouped and then sorted by number of appearances.

An idea see if we have a possibility to access original catalog item from credits list and add item names next to provider. And change the text in bottom line to `Data credits` or something similar since `Data attribution` is already taken or merge those two into one (current url can go to the bottom of modal)

**Comment by AnaBelgun** [comment]
Thanks @zoran995 there is a specific Data attribution section/link at the bottom of the screen for non-basemap data
![image](https://user-images.githubusercontent.com/18452161/182308276-1af3139d-9bb0-4bde-b987-c0f8799d9f23.png)

The basemap attribution includes links and titles of products which are are used in the making of that basemap and the confusion lies in users assuming they access/visualise a Carto/Open Street Map/Tom Tom etc product directly.

**Comment by staffordsmith83** [comment]
Is this OK @AnaBelgun and @philipgrimmett ?

![image](https://user-images.githubusercontent.com/7980991/182316663-9b15b3c1-034c-4036-a46b-7ca7d5cdb436.png)


Dear @zoran995 I havent been able to replicate where non basemap datasets are added to this `DataAttributionModal`, but I know you did some much appreciated work on this lately so let me know if Im missing something!

**Comment by zoran995** [comment]
Since #5167 all data that has attribution trait defined should have attribution value listed in the `Data attribution` (basemap) modal (it might happen that some of the attribution is not connected to cesium credit for some datasets). 
Also Cesium has internal ability to determine credits for common datasets and that is where the data attribution come (I think ESRI, Cesium ION by default have dataset credit listed), and those datasets credits where listed there even before #5167
This is default credit for bing maps aerial
![image](https://user-images.githubusercontent.com/19208948/182317259-a283d395-fe9a-4a94-aeb8-60faa97ea5be.png)

But when we add Surface geology to the map, it will also show `© Commonwealth of Australia ...` in the list
![image](https://user-images.githubusercontent.com/19208948/182317367-b4ba39b0-b3a1-4652-be6f-9c3ebc24f12a.png)

share url 
http://ci.terria.io/main/#share=s-NKaIo5aD559KCoaf8y8t4OPGCl

**Comment by staffordsmith83** [comment]
Ah I see, so now my change would result in an error..
![image](https://user-images.githubusercontent.com/7980991/182318091-0f85e40c-f348-4d34-a2ce-742c35847139.png)

Perhaps this button should not be called 'Basemap' then. What to do @AnaBelgun, to keep everyone happy?
![image](https://user-images.githubusercontent.com/7980991/182318190-48f613fe-0005-4658-8a68-85a6034c5b6f.png)

**Comment by AnaBelgun** [comment]
Data attribution is now pointing to a different page and base maps follow what Zoran described. 
Let's not make any changes atm and think about it a bit more.

**Comment by KerriG1** [comment]
Agreed @AnaBelgun  - the behaviour is more complex than original thought. Happy to descope this activity.
